docs/orleans/grains/grain-persistence/azure-cosmos-db.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Azure Cosmos DB grain persistence
+description: Use the Azure Cosmos DB grain to persist Azure Cosmos DB for NoSQL data in a .NET Orleans application.
+author: alexwolfmsft
+ms.author: alexwolf
+ms.topic: conceptual
+ms.devlang: csharp
+ms.date: 10/03/2023
+---
+
+# Azure Cosmos DB for NoSQL grain persistence
+
+The Azure Cosmos DB grain persistence provider supports the [API for NoSQL](/azure/cosmos-db/nosql).
```

Comment:
```suggestion
The Azure Cosmos DB grain persistence provider supports the [NoSQL API](/azure/cosmos-db/nosql).
```

Reply:
We have new branding guidelines for Azure Cosmos DB that are very strict about the name. We are no longer allowed to say things like "NoSQL API," "Table API," or "MongoDB API."

It's now:

- On first mention, **Azure Cosmos DB for NoSQL**
- On any further mentions, we can say the whole thing again or shorten to **API for NoSQL**